<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QDialog" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(0, 0, 0);
</string>
  </property>
  <widget class="QPushButton" name="Register_pbn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>700</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_pbn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>700</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="register_grbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>421</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(158, 158, 158);</string>
   </property>
   <property name="title">
    <string>Register</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="FName_lbl_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="FName_led_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(56, 56, 56);
border-color: rgb(170, 170, 127);
border-radius:5%;</string>
       </property>
       <property name="placeholderText">
        <string>Walter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="fname_warning_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>first name warning...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Password_lbl_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LName_led_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
border-color: rgb(170, 170, 127);
border-radius:5%;</string>
       </property>
       <property name="placeholderText">
        <string>White</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="lnam_warning_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>last name warning...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="usrname_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>User Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="usrname_led">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(56, 56, 56);
border-color: rgb(170, 170, 127);
border-radius:5%;</string>
       </property>
       <property name="placeholderText">
        <string>VoidUser78</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="usrname_warning_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>user name warning</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="LName_lbl_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="pass_led_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
border-color: rgb(170, 170, 127);
border-radius:5%;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>**********</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="confirmpass_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Confirm Pass:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="confirmpass_led">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(56, 56, 56);
border-color: rgb(170, 170, 127);
border-radius:5%;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>**********</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="password_warning_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Password warning...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="confirm_pass_warning_lbl">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>confirm Password warning...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="register_pbn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
